### PR TITLE
feat(description-list): Added description list - FRONT-2562

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/compositions/bcl-description-list/description-list.html.twig [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% spaceless %}
+
+{#
```
Comment (planctus) [inline src/compositions/bcl-description-list/description-list.html.twig:3]: i'm confused, this is called description list but this is actually a template for a single item thingy with a label and a value...
This template should be used to produce a list, and the description list i guess it refers to the html description list:https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

File: src/compositions/bcl-description-list/description-list.html.twig [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{% endif %}
+  <div class="d-flex">
+{% if _icon is not empty %}
+  <div>
+  {% include '@oe-bcl/bcl-icon/icon.html.twig' with _icon|merge({
+    attributes: create_attribute().addClass("me-2")
+  }) only %}
+  </div>
+{% endif %}
+    {{- _label -}}
```
Comment (planctus) [inline src/compositions/bcl-description-list/description-list.html.twig:36]: wouldn't it be better not to expect any markup from the user data..? couldn't we get just the "text" content..?

**Review by planctus — CHANGES_REQUESTED** [review]
I think there was a misunderstanding here :)

**Comment by planctus** [comment]
You keep forgetting to add the component/compositions you add as a dependency in bcl-twig-templates...

File: src/compositions/bcl-description-list/description-list.html.twig (hunk 1 above)
Comment (tibi2303) [reply src/compositions/bcl-description-list/description-list.html.twig:3]: Remade it the way on ecl, not sure about the extra markup for horizontal though..

File: src/compositions/bcl-description-list/description-list.html.twig (hunk 2 above)
Comment (tibi2303) [reply src/compositions/bcl-description-list/description-list.html.twig]: changed it

**Review by planctus — APPROVED** [review]

**Pull request merged**